CONTRIBUTING.md

```diff
@@ -24,70 +42,46 @@ contributing enhancements to the Loki charm.
   this charm. Please help us out in having easier reviews by rebasing
   onto the `main` branch, avoid merge commits and enjoy a linear Git
   history.
-
-
-### Setup
-
-A typical setup using [snaps](https://snapcraft.io/), for deployments to a [microk8s](https://microk8s.io/) cluster can be done using the following commands
-
-```bash
-    sudo snap install microk8s --classic
-    microk8s.enable dns storage
-    sudo snap install juju --classic
-    juju bootstrap microk8s microk8s
-    juju create-storage-pool operator-storage kubernetes storage-class=microk8s-hostpath
-```
-
-
-## Developing
-
-Create and activate a virtualenv with the development requirements:
-
-```bash
-   virtualenv -p python3 venv
-   source venv/bin/activate
-   pip install -r requirements.txt
-```
-
-
-Later on, upgrade packages as needed
-
-```bash
-   pip install --upgrade -r requirements.txt
-```
-
+- To handle our virtualenvs we use [`tox`](https://tox.wiki/en/latest/#), so there is no need
+  to deal with them manually. Tox will create, update, activate and deactivate virtualenvs for us.
 
 ### Testing
+
 All tests can be executed by running `tox` without arguments.
 
-To run individual test environments:
+Besides you can run individual test environments or tasks:
 
-```shell
-tox -e prettify  # update your code according to linting rules
-tox -e lint  # check your code complies to linting rules
-tox -e static # run static analysis
-tox -e unit  # run unit tests
+```bash
+tox -e lint         # check your code complies to linting rules
+tox -e static       # run static analysis
+tox -e unit         # run unit tests
+tox -e integration  # run integration tests
+tox -e fmt          # update your code according to linting rules
 ```
 
 Unit tests are implemented using the Operator Framework test [harness](https://ops.readthedocs.io/en/latest/#module-ops.testing).
 
 ### Build
 
-Install the [charmcraft tool](https://juju.is/docs/sdk/setting-up-charmcraft) and build the charm in this git repository:
+In order to build the charm so it can be deployed in [MicroK8s](https://microk8s.io/) using [Juju](https://juju.is/), we use [charmcraft](https://juju.is/docs/sdk/setting-up-charmcraft).
+So in the charm repository you have to run:
```

Comment:
```suggestion
In order to pack the charm locally so it could be deployed from a local path we use 
[charmcraft](https://juju.is/docs/sdk/setting-up-charmcraft).

From the charm's root folder:
```